includes/class-newspack-ui.php

```diff
@@ -33,6 +37,24 @@ public static function enqueue_styles() {
 		);
 	}
 
+	/**
+	 * Adds inline styles CSS for the element/button colors from the theme.json.
+	 * See: https://developer.wordpress.org/reference/functions/wp_get_global_styles/
+	 */
+	public static function colors_css_wrap() {
+		$global_styles = wp_get_global_styles();
+
+		$custom_css = 'body {';
```

Comment:
Other variables are under `:root`. Any specific reason why these are under `body`?

Reply:
It's the only way I could get it to work for some reason 😄 `:root` alone (or `:root` plus `!important`) for these styles didn't do the trick; I tried inserting them in a bunch of different locations in the header, but it didn't seem to make a difference. 

My guess is it works because `body` is a hair more specific than `:root`, so it's overriding the defaults; when `:root` was used, I was still getting the value set in the `_colors.scss` file.

This might have to do with how I'm inserting these styles to begin with! I tried a few different things in `<style>` tags, but maybe the cascade would be more logical if they were also enqueued in a separate file. We can definitely experiment! 🙂

Reply:
Aah, I see! Thank you for explaining 🙇